Library/Homebrew/cmd/update.sh

```diff
@@ -331,6 +345,13 @@ EOS
     UPSTREAM_BRANCH="$(upstream_branch)"
     # the refspec ensures that the default upstream branch gets updated
     (
+      if [[ -n "$HOMEBREW_UPDATE_PREINSTALL" ]]
+      then
+        # Skip taps without formulae.
+        FORMULAE="$(find "$DIR" -maxdepth 1 -name '*.rb' -or -name 'Formula' -print -quit)"
+        [ -z "$FORMULAE" ] && exit
```

Comment:
Sorry, but I'm not sure I can understand this argument. Shouldn't a variable always have the smallest possible scope that is necessary so that the code can do its job? Assuming this is something we could agree on, that scope should be reflected in the naming of the variable, where I believe we have agreed on all-caps for global and lower case for local variables. (There are some outliers, but I don't think that's a reason to be less consistent with new code.)